docs/4.2. Behaviour - Querying.md

```diff
@@ -10,25 +10,25 @@ Where the behaviour associated with an API attribute is not sufficiently clear f
 
 ### Subscriptions
 
-`secure` Indicates whether the WebSocket connection is provided via an encrypted connection (ws:// vs. wss://). Unless otherwise indicated the value of this attribute SHOULD be 'false' if the API is being presented via HTTP, and 'true' for HTTPS. Query API clients MAY choose to specify the opposite value in requests for subscriptions, however they will receive a 400 response code unless the Query API explicitly supports a mismatch between encrypted HTTP and WebSocket connections.
+`secure` Indicates whether the WebSocket connection is provided via an encrypted connection (`ws://` vs. `wss://`). Unless otherwise indicated the value of this attribute SHOULD be `false` if the API is being presented via HTTP, and `true` for HTTPS. Query API clients MAY choose to specify the opposite value in requests for subscriptions, however they will receive a 400 response code unless the Query API explicitly supports a mismatch between encrypted HTTP and WebSocket connections.
```

Comment:
Either a colon after "`secure`", or a lowercase "i" on "Indicates"?. And in the next paragraph for `authorization` too...

**Edit:** looks like that other similar paragraphs below have the colon and keep the capital.

Reply:
Colon + capital is good

Reply:
```suggestion
`secure`: Indicates whether the WebSocket connection is provided via an encrypted connection (`ws://` vs. `wss://`). Unless otherwise indicated the value of this attribute SHOULD be `false` if the API is being presented via HTTP, and `true` for HTTPS. Query API clients MAY choose to specify the opposite value in requests for subscriptions, however they will receive a 400 response code unless the Query API explicitly supports a mismatch between encrypted HTTP and WebSocket connections.
```